### DISCUSSION TITLE
Using Telescope with rg adds -L argument to rg command

### DISCUSSION THREAD

**Discussion by kritag** [LazyVim/LazyVim#4514] [Q&A] [answered]
I can run `Telescope find_files`in `$HOME` and get a few thousand hits, which i can then narrow down through fzf. 

However, I notice that when the `Lazy.pick("files")`  function is called through `<leader>ff` or `<leader>fF` if i run nvim from `$HOME` it adds the `-L`argument to the rg-command. I have the following in my telescope config, and it works (the rg-command is passed all the `-g` arguments):
```lua
        if 1 == vim.fn.executable("rg") then
          return {
            "rg",
            "--files",
            "--color",
            "never",
            "--no-follow",
            "-g",
            "!.git",
            "-g",
            "!.mozilla",
            "-g",
            "!.npm-global",
            "-g",
            "!.cache",
            "-g",
            "!.kube/cache",
            "-g",
            "!.local/share/steam",
          }
```

This is the rg-command that runs when pressing `<leader>ff/fF`:
```rg --files --color never --no-follow -g !.git -g !.mozilla -g !.npm-global -g !.cache -g !.kube/cache -g !.local/share/steam --hidden -L```

Why is `--hidden -L` added at the end here? I have searched through the LazyVim repo extensively to find out where this is set, and how to remove it. The issue with this is that it seems to loop on symlinks and therefore it never finishes. How can I have the desired behavior without resorting to swapping out `Lazy.pick("files")` for  `Telescope find_files` in my config?

**Comment by dpetka2001** [comment]
`LazyVim.pick("files", { follow = false })` should do what you want. By default it's set `follow = true`, no idea why.

**Reply by kritag** [reply]
I changed the `keys` table in my telescope config to the following:

```lua
    keys = {
      {
        --
      { "<leader>ff", LazyVim.pick("files", { follow = false }), desc = "Find Files (Root Dir)" },
      { "<leader>fF", LazyVim.pick("files", { root = false }, { follow = false }), desc = "Find Files (cwd)" },

```

but I still get the same behavior as previously mentioned that rg runs with the `-L` argument.

**Reply by dpetka2001** [reply, marked as answer]
The first one should work. The second one should be `{ root = false, follow = false }`.

**Reply by kritag** [reply]
It does now, thank you so much! :D
